src/graphnet/models/standard_model.py

```diff
@@ -62,6 +62,16 @@ def __init__(
         self._scheduler_kwargs = scheduler_kwargs or dict()
         self._scheduler_config = scheduler_config or dict()
 
+    @property
+    def target_labels(self) -> List[str]:
+        """Return target label."""
+        return [label for task in self._tasks for label in task._target_labels]
+
+    @property
+    def prediction_labels(self) -> List[str]:
+        """Return prediction labels."""
+        return [label for task in self._tasks for label in task._output_labels]
```

Comment:
The name of the member attribute should be updated.
```suggestion
        return [label for task in self._tasks for label in task._prediction_labels]
```